pulsar-client-admin-api/src/main/java/org/apache/pulsar/common/policies/data/FunctionStats.java

```diff
@@ -18,25 +18,17 @@
  */
 package org.apache.pulsar.common.policies.data;
 
-import com.fasterxml.jackson.annotation.JsonInclude;
-import com.fasterxml.jackson.annotation.JsonProperty;
-import com.fasterxml.jackson.annotation.JsonPropertyOrder;
-import java.io.IOException;
 import java.util.HashMap;
 import java.util.LinkedList;
 import java.util.List;
 import java.util.Map;
 import lombok.Data;
 import lombok.EqualsAndHashCode;
-import org.apache.pulsar.common.util.ObjectMapperFactory;
 
 /**
  * Statistics for Pulsar Function.
  */
 @Data
-@JsonInclude(JsonInclude.Include.ALWAYS)
-@JsonPropertyOrder({"receivedTotal", "processedSuccessfullyTotal", "systemExceptionsTotal", "userExceptionsTotal",
```

Comment:
is this removal changing the final output of the JSON encoded version of this structure ?

Reply:
My initial plan was to use MixIn class with jackson to add annotations but seems it is not working as I expected. So I extracted `FunctionStats` and inner classes with interfaces, please take another look on this commit https://github.com/apache/pulsar/pull/10697/commits/80fb31cab755915ef95034d2e43070f2249c130e, thanks.